src/pages/TransactionPage/index.tsx

```diff
@@ -88,8 +101,8 @@ export default function TransactionPage() {
               <RowValueColumn>{parseToFormattedNumber(tx.gasLimit)}</RowValueColumn>
             </TransactionDataRow>
             <TransactionDataRow>
-              <RowKeyColumn>Gas Used:</RowKeyColumn>
-              <RowValueColumn>0</RowValueColumn>
+              <RowKeyColumn>Transaction fee:</RowKeyColumn>
+              <RowValueColumn>{parseToFormattedNumber(transactionFee)}</RowValueColumn>
```

Comment:
Goods looks like this works!. Can you add a row now for the gas used?

Reply:
of course !